app/src/main/java/com/testtask/apptesttask/ui/characters/CharactersFragment.kt

```diff
@@ -1,21 +1,20 @@
 package com.testtask.apptesttask.ui.characters
 
+import android.widget.TextView
 import com.arellomobile.mvp.presenter.InjectPresenter
 import com.testtask.apptesttask.R
 import com.testtask.apptesttask.presentation.characters.CharactersPresenter
 import com.testtask.apptesttask.presentation.characters.CharactersView
 import com.testtask.apptesttask.ui.global.BaseFragment
-import kotlinx.android.synthetic.main.fragment_charcters.*
 
-class CharactersFragment: BaseFragment(), CharactersView {
+class CharactersFragment : BaseFragment(), CharactersView {
 
     @InjectPresenter
     lateinit var presenter: CharactersPresenter
 
-    override val layoutRes: Int
-        get() = R.layout.fragment_charcters
+    override val layoutRes = R.layout.fragment_charcters
 
     override fun showText(str: String) {
-        test_characters.text = str
+        view!!.findViewById<TextView>(R.id.test_characters).text = str
```

Comment:
`view!!.findViewById` is bad because of `findViewById` perfomance lack. You must make it `lateinit` and init into `onViewCreated`.